layers/shared/nodejs/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "dependencies": {
+    "audit": "file:../../../models/audit.js",
+    "budget": "file:../../../models/budget.js",
+    "contract": "file:../../../models/contract.js",
+    "employee-sensitive": "file:../../../models/employee-sensitive.js",
+    "employee": "file:../../../models/employee.js",
+    "expense": "file:../../../models/expense.js",
+    "expenseType": "file:../../../models/expenseType.js",
+    "tag": "file:../../../models/tag.js",
+    "trainingUrls": "file:../../../models/trainingUrls.js",
+    "databaseModify": "file:../../../js/databaseModify.js",
+    "dateUtils": "file:../../../js/dateUtils.js",
+    "GetUserInfoMiddleware": "file:../../../js/GetUserInfoMiddleware.js",
+    "Logger": "file:../../../js/Logger.js",
+    "utils": "file:../../../js/utils.js",
+    "crudRoutes": "file:../../../routes/crudRoutes.js",
+    "expenseRoutes": "file:../../../routes/expenseRoutes.js"
+  }
```

Comment:
In the https://stackoverflow.com/questions/60572458/how-do-i-use-shared-code-in-lambdas-in-an-aws-sam-template-using-layers-in-node post, it mentions:

> Gotchas
> 
> The following gotcha has been avoided in this solution. I am mentioning it because it looked like a straight-forward solution and it took a lot of time before I finally abandoned it.
> 
> It is very tempting to add a folder reference in the lambda function's package.json.
> 
> //...
> "dependencies": {
>     "common":"file:../../layers/layer1/nodejs/common"
> },
> //...
> If you do that, it will work the first sam build. However, the second time you run sam build, your shared code folder and all subdirectories will be DELETED. This is because when sam builds, it creates an .aws-sam folder. If that folder exists, it performs an npm cleanup, and I think that is what provokes the deleting of the shared code.

Do we need to worry about code being deleted during the second deployment?